pkg/kubectl/bind/plugin/bind.go

```diff
@@ -55,8 +55,9 @@ type BindOptions struct {
 	*base.Options
 	Logs *logs.Options
 
-	Print  *genericclioptions.PrintFlags
-	DryRun bool
+	Print   *genericclioptions.PrintFlags
+	Printer printers.ResourcePrinter
```

Comment:
nit: make is lower-case, or put a `if b.Printer == nil` around the line setting it below.